notebooks/108-gpu-device/108-gpu-device.ipynb

```diff
@@ -0,0 +1,1009 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eadfb30e-d6d3-4ed3-8e7e-f1a843ce2ec0",
+   "metadata": {},
+   "source": [
+    "### Compile the Model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8648ae4f-2c73-4525-ace0-1c71afb18ec4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Read model and compile it on GPU in THROUGHPUT mode\n",
+    "model = core.read_model(model=model_path)\n",
+    "device_name = \"GPU\"\n",
+    "compiled_model = core.compile_model(model=model, device_name=device_name, config={\"PERFORMANCE_HINT\": \"THROUGHPUT\"})\n",
+    "\n",
+    "# Get the input and output nodes\n",
+    "input_layer = compiled_model.input(0)\n",
+    "output_layer = compiled_model.output(0)\n",
+    "\n",
+    "# Get the input size\n",
+    "num, height, width, channels = input_layer.shape\n",
+    "print('Model input shape:', num, height, width, channels)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "083bce3d-647d-46d1-8540-4a2935bc829f",
+   "metadata": {},
+   "source": [
+    "### Load and Preprocess Video Frames "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fcee85d9-eab1-4a94-bb2f-d5217fd7cdcf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load video\n",
+    "video_file = \"../data/video/Coco Walking in Berkeley.mp4\"\n",
+    "video = cv2.VideoCapture(video_file)\n",
+    "framebuf = []\n",
+    "\n",
+    "# Go through every frame of video and resize it\n",
+    "print('Loading video...')\n",
+    "while video.isOpened():\n",
+    "    ret, frame = video.read()\n",
+    "    if not ret:\n",
+    "        print('Video loaded!')\n",
+    "        video.release()\n",
+    "        break\n",
+    "    \n",
+    "    # Preprocess frames - convert them to shape expected by model\n",
+    "    input_frame = cv2.resize(src=frame, dsize=(width, height), interpolation=cv2.INTER_AREA)\n",
+    "    input_frame = np.expand_dims(input_frame, axis=0)\n",
+    "\n",
+    "    # Append frame to framebuffer\n",
+    "    framebuf.append(input_frame)\n",
+    "    \n",
+    "\n",
+    "print('Frame shape: ', framebuf[0].shape)\n",
+    "print('Number of frames: ', len(framebuf))\n",
+    "\n",
+    "# Show original video file\n",
+    "# If the video does not display correctly inside the notebook, please open it with your favorite media player\n",
+    "Video(video_file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7003286a-8349-46f6-9d38-c6bb959e892a",
+   "metadata": {},
+   "source": [
+    "### Define Model Output Classes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "621b919e-c120-42f9-89eb-ffd7b810c91f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define the model's labelmap (this model uses COCO classes)\n",
+    "classes = [\n",
+    "    \"background\", \"person\", \"bicycle\", \"car\", \"motorcycle\", \"airplane\", \"bus\", \"train\",\n",
```

Comment:
probably we can consider to  have some label map for coco dataset in notebooks/data for avoid coping these from tutorial to tutorial